### PR TITLE
PYIC-3507: add stub data for NINO cri

### PR DESCRIPTION
**Pull request by shivanshuit914**
## Proposed changes

- Add stub data for the NINO cri verification.

### Issue tracking

- [PYI-3507](https://govukverify.atlassian.net/browse/PYI-3507)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json [hunk 1]
```diff
@@ -1302,6 +1302,40 @@
           }
         ]
       }
+    },
+    {
+      "criType": "National Insurance Number Verification (Stub)",
+      "label": "Alice Parker (Valid) National Insurance",
```
Comment (blakeyp) [inline di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json:1308]: Can we add ones also for Kenneth D, Mary Watson, Bob Parker .. the full house 😄

File: di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json [hunk 2]
```diff
@@ -1302,6 +1302,40 @@
...
+            "value": "1970-01-01"
+          }
+        ],
+        "socialSecurityRecord": [
+          {
+            "personalNumber": "AA000003D"
+          }
+        ]
+      }
     }
```
Comment (blakeyp) [inline di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json:1429]: Also can we add a success and fail evidence block in the criStubEvidencePayloads.json file - like the examples here: https://docs.google.com/document/d/19MssTJO1JxmHw_pYrRNkSvwfQdFdxuoEtx0m0uqKbsE/edit#heading=h.6tqvu5u4vw5l
Should just have checkDetails/failedCheckDetails (no scores)

File: di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json (hunk 1 above)
Comment (shivanshuit914) [reply di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json:1308]: Done, Thanks

File: di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json (hunk 2 above)
Comment (shivanshuit914) [reply di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json:1429]: Done, Thanks

**Review by blakeyp — APPROVED** [review]

**Pull request merged**